index.html

```diff
@@ -39,17 +39,17 @@ <h2>My Three Favourite Hobies</h2>
 			</li>
 		</ol>
 	<h2>Haiku</h2>
-		<p>The chill, worming in</p>
-		<p>Shock, pleasure, bursting within</p>
-		<p>Summer tongue awakes</p>
-		<p>By <b>Suh Joon Kim</b></p>
+		<p>The chill, worming in<br>
+		Shock, pleasure, bursting within<br>
+		Summer tongue awakes<br>
+		<em>By <b>Suh Joon Kim</b></em></p>
```

Comment:
To create line breaks within a paragraph, use `<br>`.  The `<p>` element should only be used to separate different paragraphs.